.gitlab-ci.yml

```diff
@@ -190,3 +192,28 @@ force_release_prod:
   only:
     - staging
 
+verify_sync_partial:on-schedule:
```

Comment:
Where are the scheduling details set for the verify_sync tasks?

Reply:
They must be set in Gitlab - to my understanding, they can't be set in `.gitlab-ci.yml`

Reply:
I would suggest adding a one line comment that mentions that

Reply:
I think i was wrong in thinking it could only be set in the UI, https://docs.gitlab.com/ee/user/project/pipelines/schedules.html

Reply:
We might want to differentiate scheduled by stage, I think it would be a waste to run it on dev; but as we advance through the stages we should have more frequent checks. integration every 10 days, staging 7 days, prod 3 days.